src/analysis/convex/between.lean

```diff
@@ -378,6 +378,28 @@ lemma sbtw.trans_right [no_zero_smul_divisors R V] {w x y z : P} (h₁ : sbtw R
   (h₂ : sbtw R x y z) : sbtw R w y z :=
 h₁.wbtw.trans_sbtw_right h₂
 
+lemma wbtw.trans_left_ne [no_zero_smul_divisors R V] {w x y z : P} (h₁ : wbtw R w y z)
+  (h₂ : wbtw R w x y) (h : y ≠ z) : x ≠ z :=
```

Comment:
This to me reads like you're doing transitivity on the rightmost two elements, so it should rather be
```suggestion
lemma wbtw.trans_right_ne [no_zero_smul_divisors R V] {w x y z : P} (h₁ : wbtw R w x y)
  (h₂ : wbtw R w y z) (h : y ≠ z) : x ≠ z :=
```
but I see you already have been using another in the rest of the file.